### PR TITLE
fix quantity, craft command, and help window

### LINKED ISSUE #104: [PE-D] Help Command leads to markdown file instead of github website
I think this is a small issue but its worth noting for your final PE

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: GabrielSimbingyang/ped#1

### LINKED ISSUE #115: [PE-D] craft: No reaction from high quantity input
No reaction from high quantity input

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: GabrielSimbingyang/ped#12

### LINKED ISSUE #125: [PE-D] Cannot refresh bug
After entering an invalid command, the system stucks at the command and cannot proceed. For example, if we use an invalid addr command, then when we enter another addq command, the system freezes

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: raymondge/ped#4

### LINKED ISSUE #136: [PE-D] addi command: quantity display
It is a very minor issue, but I think it would be better to display 1 instead of 00001

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: YangYue128-helen/ped#4

### PR DESCRIPTION
**Pull request by justacasul**
bruh why you breaking aBStRaCtiON by using int in changing quantity

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Sun Nov 01 21:40:16 SGT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```
Comment (aqcd) [inline gradle/wrapper/gradle-wrapper.properties]: concerning, any reason why?

**Review by aqcd — COMMENTED** [review]
link to issues this is resolving pls

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (justacasul) [reply gradle/wrapper/gradle-wrapper.properties]: Huh might be because I deleted everything and git cloned the whole tp again

Comment (justacasul) [reply gradle/wrapper/gradle-wrapper.properties]: Idk what difference this makes lmao

Comment (aqcd) [reply gradle/wrapper/gradle-wrapper.properties]: try reverting this particular chunk and see if you can still run?

**Pull request merged**